src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -173,23 +173,13 @@
       (style/create-section-header "Analysis Submissions")
       (style/create-paragraph
         (let [count-all (count submissions)
-              success-count (->> submissions (filter all-success?) count)
```

Comment:
I was unable to find the original motivation for this logic.  I saw @rushtong and @MattPutnam in the change history here, so I would appreciate any recollection you have of this.